cmd/instance_pool_list.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type instancePoolItem struct {
+	ID    string                     `json:"id"`
+	Name  string                     `json:"name"`
+	Zone  string                     `json:"zone"`
+	Size  int                        `json:"size"`
+	State egoscale.InstancePoolState `json:"state"`
+}
+
+type instancePoolListItemOutput []instancePoolItem
+
+func (o *instancePoolListItemOutput) toJSON()  { outputJSON(o) }
+func (o *instancePoolListItemOutput) toText()  { outputText(o) }
+func (o *instancePoolListItemOutput) toTable() { outputTable(o) }
+
+type outputResult struct {
```

Comment:
This type name is way too vague, keep in mind that you're operating in a global namespace: purpose-specific types such as this one have to be prefixed to both hint at what they're really meant for and to avoid collisions in other parts of the namespace.

```suggestion
type instancePoolFetchResult struct {
```

Reply:
thx